pkg/services/provisioning/provisioning.go

```diff
@@ -144,34 +148,28 @@ func (ps *provisioningServiceImpl) ProvisionNotifications() error {
 }
 
 func (ps *provisioningServiceImpl) ProvisionDashboards() error {
-	dashboardPath := filepath.Join(ps.Cfg.ProvisioningPath, "dashboards")
-	dashProvisioner, err := ps.newDashboardProvisioner(dashboardPath, ps.SQLStore)
-	if err != nil {
-		return errutil.Wrap("Failed to create provisioner", err)
-	}
-
+	// TODO read configs when provisioning using the disk service.
```

Comment:
:warning: I changed a bit the behavior.

This method is called by a "reload" endpoint. 
Previously, when we had a "reload", this would create a new dashboard provisioner (which would read all provisioning config file from disk). Now we keep the same service and just call the Provision method straight away.
I need to fix this.

Reply:
OK let's keep the TODO for now? 😇